### PR TITLE
[OC-1464] Fix bug when deleting all bundles

### PR DESCRIPTION
**Pull request by AlexGuironnetRTE**
For release notes, under "Bugs":

[OC-1464] Fix bug when deleting all bundles

[OC-1464]: https://opfab.atlassian.net/browse/OC-1464

### CODE REVIEW

File: services/core/businessconfig/src/test/java/org/lfenergy/operatorfabric/businessconfig/controllers/GivenNonAdminUserBusinessconfigControllerShould.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import org.lfenergy.operatorfabric.springtools.configuration.test.WithMockOpFabUser;
 import org.lfenergy.operatorfabric.businessconfig.application.IntegrationTestApplication;
```
Comment (quinarygio) [inline services/core/businessconfig/src/test/java/org/lfenergy/operatorfabric/businessconfig/controllers/GivenNonAdminUserBusinessconfigControllerShould.java:17]: Fix Copyright

File: services/core/businessconfig/src/test/java/org/lfenergy/operatorfabric/businessconfig/services/ProcessesServiceShould.java [hunk 2]
```diff
@@ -16,7 +16,6 @@
 import static org.lfenergy.operatorfabric.businessconfig.model.ResourceTypeEnum.I18N;
```
Comment (quinarygio) [inline services/core/businessconfig/src/test/java/org/lfenergy/operatorfabric/businessconfig/services/ProcessesServiceShould.java:16]: Fix copyright

**Review by quinarygio — CHANGES_REQUESTED** [review]

File: services/core/businessconfig/src/test/java/org/lfenergy/operatorfabric/businessconfig/controllers/GivenNonAdminUserBusinessconfigControllerShould.java (hunk 1 above)
Comment (AlexGuironnetRTE) [reply services/core/businessconfig/src/test/java/org/lfenergy/operatorfabric/businessconfig/controllers/GivenNonAdminUserBusinessconfigControllerShould.java:17]: Done

File: services/core/businessconfig/src/test/java/org/lfenergy/operatorfabric/businessconfig/services/ProcessesServiceShould.java (hunk 2 above)
Comment (AlexGuironnetRTE) [reply services/core/businessconfig/src/test/java/org/lfenergy/operatorfabric/businessconfig/services/ProcessesServiceShould.java:16]: Done

**Review by quinarygio — APPROVED** [review]

**Pull request merged**